wiki/versions/1.19/getting_started/setup/markdown/migrating_from_fabric.md

```diff
@@ -0,0 +1,48 @@
+# Migrating From Fabric
+
+While Quilt supports loading Fabric mods, you might want to experience some additional features that Quilt might
+provide.
+
+To set up a Quilt mod workspace, check the [`Setup` guide](./setup).
+
+## Notable Differences
+
+Although many things are very similar, there are a few remarkable divergences between Quilt and Fabric mods.
+
+### Mod Metadata
+
+Quilt utilizes a `quilt.mod.json` which is largely different from the `fabric.mod.json`.
+More information can be found at the [quilt.mod.json specification RFC](https://github.com/QuiltMC/rfcs/blob/main/specification/0002-quilt.mod.json.md).
```

Comment:
Advertise the [QMJ JSON schema](https://raw.githubusercontent.com/QuiltMC/quilt-json-schemas/main/quilt.mod.json/schemas/main.json) here! It might be useful for anyone who wants to make sure they're doing things right